test/nootils.spec.js

```diff
@@ -208,4 +208,25 @@ describe('Utils', () => {
       expect(actual).to.eq(false);
     });
   });
+
+  describe('isFormSubmittedInWindow', () => {
+    it('returns false if no matching report in window', () => {
+      const reports = [
+        { form: 'A', reported_date: 1 }
+      ];
+      const actual = nootils.isFormSubmittedInWindow(reports, 'A', 2, 3);
+      expect(actual).to.eq(false);
+    });
+
+    it('returns true if there are matching reports in window', () => {
+      const reports = [
+        { form: 'A', reported_date: 1 },
+        { form: 'A', reported_date: 1 },
+        { form: 'B', reported_date: 1 },
+        { form: 'C', reported_date: 1 }
+      ];
+      const actual = nootils.isFormSubmittedInWindow(reports, ['A', 'B'], 1, 3);
+      expect(actual).to.eq(true);
+    });
```

Comment:
Please also add a unit test when you pass a matching report and a string `forms` parameter.